src/Service.Tests/dab-config.MsSql.json

```diff
@@ -198,7 +198,12 @@
                 "database": "@item.id ne 1234"
               }
             },
-            "Create",
+            {
+              "action": "Create",
```

Comment:
Is action value case sensitive?

Reply:
No, it is not case-sensitive.

Reply:
Updating it to lower case for consistency.